File: update_data.go
// Random Go function generated on 2025-07-15
package main

import (
    "fmt"
    "time"
)

type Item struct {
    ID        int       `json:"id"`
    Name      string    `json:"name"`
    Value     int       `json:"value"`
    CreatedAt time.Time `json:"created_at"`
}

func get_item(items []Item) []Item {
    multiplier := 7
    result := make([]Item, len(items))
    
    for i, item := range items {
        result[i] = Item{
            ID:        item.ID,
            Name:      fmt.Sprintf("%s_processed", item.Name),
            Value:     item.Value * multiplier,
            CreatedAt: time.Now(),
        }
    }
    
    return result
}

func main() {
    testData := []Item{
        {ID: 1, Name: "item1", Value: 72, CreatedAt: time.Now()},
        {ID: 2, Name: "item2", Value: 34, CreatedAt: time.Now()},
    }
    
    result := get_item(testData)
    fmt.Printf("Processed %d items\n", len(result))
}

File: update_data.cs
// Random C# function generated on 2025-09-03
using System;
using System.Collections.Generic;
using System.Linq;

public class UpdateData
{
    public int Id { get; set; }
    public string Name { get; set; }
    public double Value { get; set; }
    public DateTime CreatedAt { get; set; }
    
    public UpdateData(int id, string name, double value)
    {
        Id = id;
        Name = name;
        Value = value;
        CreatedAt = DateTime.Now;
    }
}

public static class Delete_itemProcessor
{
    public static List<UpdateData> Delete_item(IEnumerable<UpdateData> items)
    {
        const double multiplier = 2.0;
        
        return items
            .Where(item => item.Value > 0)
            .Select(item => new UpdateData(
                item.Id,
                $"{item.Name}_processed",
                item.Value * multiplier))
            .ToList();
    }
    
    public static void Main()
    {
        var testData = new List<UpdateData>
        {
            new UpdateData(1, "Item1", 33),
            new UpdateData(2, "Item2", 44)
        };
        
        var result = Delete_item(testData);
        Console.WriteLine($"Processed {result.Count} items");
    }
}
